### PR TITLE
More fix for Vg and VG ##visual

### PR DESCRIPTION
**Pull request by Vane11ope**
For example, s 0 -> Vg nor VG works so fixed.

### CODE REVIEW

**Comment by radare** [comment]
asuming head/last are not null and are sorted is a wrong assumption

File: libr/core/visual.c [hunk 1]
```diff
@@ -2740,11 +2740,13 @@ R_API int r_core_visual_cmd(RCore *core, const char *arg) {
 			break;
 		case 'g':
 			if (core->io->va) {
-				ut64 offset = r_io_section_get_paddr_at (core->io, 0LL);
-				if (offset == -1) {
-					offset = 0;
-				}
 				RIOMap *map = r_io_map_get (core->io, core->offset);
+				if (!map) {
+					map = ls_pop (core->io->maps);
```
Comment (radare) [inline libr/core/visual.c]: use ls_tail instead of pop/push

File: libr/core/visual.c [hunk 2]
```diff
@@ -2757,6 +2759,12 @@ R_API int r_core_visual_cmd(RCore *core, const char *arg) {
 			ret = -1;
 			{
 				RIOMap *map = r_io_map_get (core->io, core->offset);
+				if (!map) {
```
Comment (radare) [inline libr/core/visual.c]: just ls_head()->data

**Comment by Vane11ope** [comment]
done.

**Pull request merged**